src/Framework/TestSuite.php

```diff
@@ -602,56 +602,56 @@ public function run(TestResult $result = null): TestResult
 
         $test = null;
 
-        try {
-            foreach ($hookMethods['beforeClass'] as $beforeClassMethod) {
-                if ($this->testCase &&
-                    class_exists($this->name, false) &&
-                    method_exists($this->name, $beforeClassMethod)) {
-                    if ($missingRequirements = TestUtil::getMissingRequirements($this->name, $beforeClassMethod)) {
-                        $this->markTestSuiteSkipped(implode(PHP_EOL, $missingRequirements));
-                    }
+        if ($this->testCase && class_exists($this->name, false)) {
```

Comment:
Unless I'm missing something, there does not appear to be any code executed within the loops that could change the outcome of this condition, or is there?